CDCgov/prime-reportstream
# Adds fix for the comment not carrying over on inbound HL7 messages

This PR fixes an issue with the NTE segments not carrying over when ingesting HL7. It walks the body of an HL7 message and collates all of the different NTE segment values into a single message that is then put into an NTE segment associated with the first OBX segment in the output HL7 document. The content is trimmed to the length of the `FTE` data type in HL7, which 65,536 characters.

Test Steps:
1. Run `gradle test testIntegration`
2. Run `./prime test`
3. Generate a test file that contains multiple comment sections and for target state of MN or TN
4. Submit the file to your local Docker container for a monkeypox sender
5. Examine the output in your local `build/sftp` folder
6. Confirm it has collated the NTE comments into a single NTE segment associated with the first OBX segment

## Changes
- Schema update to remove duplicated NTE-3 output field
- Method to pull all the NTE segments into a single comment
- Unit tests!

## Checklist

### Testing
